### DISCUSSION TITLE
Unity debugging support

### DISCUSSION THREAD

**Discussion by tolosafan** [zed-industries/zed#8618] [Feature Requests]
Hi, does Zed work to debug a Unity project (C#) on MacOS?

**Comment by Moshyfawn** [comment]
You can follow Zed's debugging capabilities in #5065.

At the moment, Zed doesn't have any debugging capabilities.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
